README.md

```diff
@@ -366,7 +366,7 @@ Please [discover modm's peripheral drivers for your specific device][discover].
 <td align="center">✅</td>
 <td align="center">✅</td>
 <td align="center">✗</td>
-<td align="center">✗</td>
+<td align="center">○</td>
```

Comment:
Huh. This is actually an obscure bug that took me a while to dig into. Not related to my change.

TL;DR: due to a bug in parsing stm32 clock module, all devices with flash module will have this one selected too. I'll file a separate bug for that.

Reply:
Filled as https://github.com/modm-io/modm/issues/671

Since I fixed the flash tick, this problem is not applicable to my PR anymore.